optin/puller/workflows/payout.py

```diff
@@ -51,8 +51,7 @@ async def process_unprocessed_requests(unprocessed_requests, sent_requests, data
 			if databases.multisig.check_cosigners(address, [request.address for request in request_group.requests]):
 				balance = databases.balances.lookup_balance(address)
 				symbol_address = symbol_network.public_key_to_address(request_group.requests[0].destination_public_key)
-				nem_transaction_hash = request_group.requests[0].transaction_hash
-				sent_transaction_hash = send_funds(symbol_address, balance, nem_transaction_hash)
+				sent_transaction_hash = send_funds(symbol_address, balance, address)
```

Comment:
and here I hope address in grouping is what we want to be in that message, right?